### PR TITLE
Fobnail general description

### PR DESCRIPTION
**Pull request by Asiderr**
Signed-off-by: Norbert Kamiński <norbert.kaminski@3mdeb.com>

### CODE REVIEW

File: docs/description.md [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+https://www.usenix.org/legacy/event/hotsec07/tech/full_papers/mccune/mccune_html/index.html
+
+[2] "Remote Attestation Procedures Architecture",
+H. Birkholz, D. Thaler, M. Richardson, N. Smith, W. Pan,
+https://datatracker.ietf.org/doc/draft-ietf-rats-architecture/
+
+[3] "Fobnail Token provisioning - Fobnail architecture", M. Zmuda, N. Kamiński,
+https://drive.google.com/file/d/143wW4zWcLbedbuaWMlQpC9f_CMCcXdLH/view?usp=sharing
+
+[4] "External platform provisioning - Fobnail architecture", M. Zmuda, N. Kamiński,
```
Comment (macpijan) [inline docs/description.md:87]: @Asiderr Wouldn't it be more readable if we would put images directly to let them render in a document?

Comment (macpijan) [reply docs/description.md:87]: We have uploaded some PNG images but they are not used anywhere in text?

Comment (Asiderr) [reply docs/description.md:87]: @macpijan I'm not sure if we want to show fobnail diagrams in the project description. It's described and shown in the Fobnail architecture.

Comment (Asiderr) [reply docs/description.md:87]: @macpijan It is used in the [architecture.md](https://github.com/fobnail/docs/pull/1/files#diff-140eef3ba41bdcf401d507408084181f2c0ac627532b61e0f7906ea7cc926782R33)

**Pull request merged**